core/src/main/java/fr/sncf/osrd/stdcm/graph/STDCMEdgeBuilder.java

```diff
@@ -128,7 +129,8 @@ public Collection<STDCMEdge> makeAllEdges() {
                     graph.comfort,
                     graph.timeStep,
                     STDCMUtils.getStopsOnRoute(graph, route, startOffset),
-                    graph.tag
+                    graph.tag,
+                    new DriverBehaviour()
```

Comment:
This still changes stdcm behavior.

Please keep changes in stdcm files minimal. You can run this command to reset it:
 
    git checkout dev -- core/src/test/java/fr/sncf/osrd/stdcm core/src/main/java/fr/sncf/osrd/stdcm core/src/main/java/fr/sncf/osrd/api/stdcm

After that, there are only a few calls to `StandaloneSim.run` to fix, one in the endpoint and one in the test. The values there need to be (0, 0).